backend/tests/test_assignment_double_count.py

```
@@ -0,0 +1,310 @@
+
+@pytest.fixture()
+def db_session():
+    """In-memory SQLite session for the import + recompute pipeline."""
+    engine = create_engine(
+        "sqlite:///:memory:",
+        connect_args={"check_same_thread": False},
+        poolclass=StaticPool,
+    )
+
+    @event.listens_for(engine, "connect")
+    def _set_pragma(dbapi_conn, connection_record):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_set_pragma`

Add return type annotation: `None`

(ANN202)